app/applications-tab/catalog/route.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
+    return Ember.$.ajax('/catalog/templates', 'GET').then((response) => {
+      var data = _.sortBy(response.data, (i) => {
```

Comment:
Is there any reason to use `_` in this function vs the built-in Ember `array.sortBy('name')` (which does `localeCompare` and is case-insensitive-ish), `array.filterBy('category', params.type)` and `array.forEach`?

`_` sort of only exists because one of the other libraries needs it.